src/types/union.ts

```diff
@@ -178,7 +178,7 @@ function analyzePossibleDiscriminators(
     return found ? [...found.values()] : [];
 }
 
-function propsInfoToProps(propsInfo: PropertiesInfo): Properties {
+export function propsInfoToProps(propsInfo: PropertiesInfo): Properties {
```

Comment:
This one I'm using in pick now, but also don't know if it's okay to export just like that.